pyintegration/deephaven2/table_listener.py

```diff
@@ -0,0 +1,270 @@
+#
```

Comment:
Should this module be named `table_listener`, `listener`, or should it be part of the `table` module?

Reply:
I like table_listener because it is clear, accurate. 'listener' is a bit too broad. Folding it into the table module would make it a bit bloated and arguably less cohesive.